text/user.css

```
@@ -956,20 +953,36 @@ button[data-testid="control-button-repeat"][aria-checked="mixed"]
-    )
-    > span::after {
-    content: "\21AA";
+button[data-testid="control-button-seek-forward-15"] > span:first-child::after {
+    content: "+15s";
+    white-space: nowrap;
+}
+/* podcast controls */
+button[data-testid="control-button-playback-speed"] > span.hidden-visually {
+    width: auto !important;
+    height: auto !important;
+    clip: auto !important;
```

Stylelint (17.6.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)